internal/scaffold/olm-catalog/csv.go

```diff
@@ -234,6 +234,31 @@ func (s *CSV) setCSVDefaultFields(csv *olmapiv1alpha1.ClusterServiceVersion) {
 			{Type: olmapiv1alpha1.InstallModeTypeAllNamespaces, Supported: true},
 		}
 	}
+	if len(csv.Spec.Icon) == 0 {
+		csv.Spec.Icon = []olmapiv1alpha1.Icon{
+			olmapiv1alpha1.Icon{
+				Data:      "Placeholder data",
```

Comment:
If we're going to add placeholder text, I'd like to see it capitalized to make it obvious and regexp-able, ex. `PLACEHOLDER_DATA`.

Reply:
I followed the current standard, so, we need to replace in all places. I will do it now.